test/test-applications/regression/HttpMessageHandler.StackOverflowException/HttpMessageHandler.StackOverflowException.csproj

```diff
@@ -1,4 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <LoadManagedProfilerFromProfilerDirectory>true</LoadManagedProfilerFromProfilerDirectory>
```

Comment:
Why does this app this need property but not other apps?

Reply:
This property is needed to copy the automatic instrumentation to the `profiler-lib` directory when the app is built. At this point, I think we can do some refactoring so that this file-copying behavior is the default, but we still need the property at the time of this commit

Reply:
But why do we need it in this specific app only? What's special about `HttpMessageHandler.StackOverflowException`?

Reply:
Most other apps have this, so it's not unique. Without the property, we just wouldn't be testing automatic instrumentation when running this locally.

But I just noted that there are other projects that reference Datadog.Trace but don't have the property, so I'm going to make suggestions to fix that.

Reply:
The weird thing is that _without_ this, the samples blow up with: 

```
Unhandled exception. System.IO.FileNotFoundException: Could not load file or assembly 'Datadog.Trace, Version=1.28.5.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb'. The system cannot find the file specified.
```

This is what happened on the `void main` samples, but it happens on this one too for some unknown reason...

Reply:
Andrew, can you clarify the scenario? I fully expect that to happen if
1. `<ProjectReference Include="..\..\..\..\src\Datadog.Trace\Datadog.Trace.csproj" Private="false" />`
2. Property `LoadManagedProfilerFromProfilerDirectory` is not set

And the reason for that is that the Datadog.Trace.dll is now no longer in the app output (`Private="false"`) and with `LoadManagedProfilerFromProfilerDirectory` being unset, the automatic instrumentation does not happen so there's no ALC fallback to find Datadog.Trace.dll

Reply:
I'm speaking for @kevingosse here, but I'm talking about the samples blowing up in CI, when we have automatic instrumentation enabled. I don't see why this sample is any different from other samples we run in CI which don't have this property set.

As an aside, this sample previously wasn't failing if the profiler was attached, and we don't check spans are generated, so it theoretically might not have actually been testing anything (if auto instrumentation wasn't enabled for some reason)